html-css/week1/Suman-Order-Desktop/styles.css

```
@@ -0,0 +1,318 @@
+body {
+  margin: 0 auto;
+  font-family: Arial;
+}
+
+body a {
+  cursor: pointer;
+}
+
+/* --- Header --- */
+.order {
+  color: white;
+  display: flex;
+  flex-direction: column;
+  background: linear-gradient(92.74deg, #FD7B46 1.15%, #FE4E9B 98.48%);
+  padding: 2%;
+}
+
+.order_summary {
+  align-self: center;
+}
+
+.order_total {
+  display: flex;
+  flex-direction: column;
+}
+
+.order_totalpermonth {
+  display: flex;
+  justify-content: space-between;
+  opacity: 0.8;
+}
+
+.order_totalnow {
+  display: flex;
+  justify-content: space-between;
+  font-weight: 700;
+}
+
+.order_line {
+  border: 0;
+  clear: both;
+  display: block;
+  background: white;
+  height: 2px;
+  width: 99%;
+  margin: -6px 0 -6px 0;
+}
+
+/* --- Payment Method --- */
+.payment {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.payment_method {
+  align-self: center;
+  padding-top: 24px;
+}
+
+.payment_option {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 20px;
+}
+
+.payment_button {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #939393;
+  width: 47%;
+  gap: 10px;
+  font-size: 12px;
+  font-weight: 700;
+  font-family: Montserrat;
+  height: 110px;
+  border: transparent;
+  background: #FFFFFF;
+  box-shadow: 8px 8px 20px rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
+}
+
+.payment_button:active {
+  border: 4px solid #fe4e9b;
+}
+
+.payment_button:focus {
+  border: 4px solid #fe4e9b;
+}
+
+.payment_button:target {
+  border: 4px solid #fe4e9b;
+}
```

This can be written like this:
```suggestion
.payment_button:active,
.payment_button:focus-visible,
.payment_button:target {
  border: 4px solid #fe4e9b;
}
```

`Focus-visible` is preferred than just `focus` as it appears only on keyboard interaction and not click. Click is already covered by the `hover` and `active` style.